sdk/communication/azure-communication-sms/README.md

```diff
@@ -161,7 +161,8 @@ try {
 
 ## Next steps
 
-Check out other client libraries for Azure Communication Services
+- [Read more about SMS in Azure Communication Services][nextsteps]
```

Comment:
Lets keep the naming convention consistent with the rest. `nextsteps` -> `next_steps`